source/includes/candidate-ingestion/_prospect_pools.md

```diff
@@ -0,0 +1,84 @@
+# Prospect Pools
+
+## Retreive Prospect Pools
+
+```shell
+curl 'https://api.greenhouse.io/v1/partner/prospect_pools'
+-H "Authorization: Basic MGQwMzFkODIyN2VhZmE2MWRjMzc1YTZjMmUwNjdlMjQ6"
+```
+> API Response
+
+```json
+[
+    {
+        "id": 123,
+        "name": "Cold Outreach: Referred",
+        "stages": [
+            {
+                "id": 1957,
+                "name": "Not Contacted",
+            },
+            {
+                "id": 1958,
+                "name": "Contacted",
+            },
+            {
+                "id": 1959,
+                "name": "1st. Follow Up Sent",
+            },
+            {
+                "id": 1960,
+                "name": "2nd Follow Up Sent",
+            },
+            {
+                "id": 1961,
+                "name": "In Discussion",
+            }
+        ]
+    },
+    {
+        "id": 456,
+        "name": "Cold Outreach: Sourced",
+        "stages": [
+            {
+                "id": 1962,
+                "name": "Not Contacted",
+            },
+            {
+                "id": 1963,
+                "name": "Contacted",
+            },
+            {
+                "id": 1964,
+                "name": "1st. Follow Up Sent",
+            },
+            {
+                "id": 1965,
+                "name": "2nd Follow Up Sent",
+            },
+            {
+                "id": 1966,
+                "name": "In Discussion",
+            }
+        ]
+    }
+]
+```
+
+Retrieve prospect pools along with their stages for the current user's organization.
+
+`GET https://api.greenhouse.io/v1/partner/prospect_pools`
```

Comment:
Should we add a note that stages will be returned in the order they should be displayed to the user?

Reply:
@davidhan527 - Did you see this one?  Just clarifying you're not doing this before I merge.

Reply:
Yeah I made that change in the description here - https://github.com/grnhse/greenhouse-api-docs/pull/151/commits/3c17aa0b231bcaa3a6aa2cda4bd89ec6f7fee807#diff-08b6358a993e825b70e3384c94d00bb9R177

Thanks for checking.

Reply:
👍